src/apps/material/material.test.ts

```diff
@@ -41,7 +41,7 @@ describe("Material", () => {
 
     cy.visit("/iframe.html?id=apps-material--default&viewMode=story&type=bog");
 
-    cy.getBySel("material-description-series-1")
+    cy.getBySel("material-description-series-0")
```

Comment:
This test specifically tests for series. Consider updating the title to reflect that.